data_ingestion.ipynb

```diff
@@ -22,7 +22,7 @@
         "colab_type": "text"
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #24.    default_values = {
</pre><p>Are these the same values for TimberID as we use in the CSVs on Google Drive? You can preview the TimberID output at <a href="https://app.gitbook.com/o/QBMPp7Gm0T0PO9PvQkUz/s/kD1zcJTeGwRpwjArj78G/~/changes/20/architecture-of-timberid/detailed-design/research-colabs/data-ingestion" target="_blank">https://app.gitbook.com/o/QBMPp7Gm0T0PO9PvQkUz/s/kD1zcJTeGwRpwjArj78G/~/changes/20/architecture-of-timberid/detailed-design/research-colabs/data-ingestion</a> and there are some notable differences:</p><ul><li>The TimberID datasource has both "lon" and "long", and I'm not sure which the downstream models use (or whether they need both)</li><li>TimberID does not provide Iso_Oxi_Stack_, predkrig_br_, brisoscape_mean_, or isoscape_fullmodel_ features</li><li>TimberID has some additional features that I don't see used here</li></ul><p> </p><p>It may be worth noting these differences.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/tnc-br/ddf-isoscapes/pull/191/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>

Reply:
When the columns are not present, they'll be ignored by the data ingestion. What this means here is that the TimberID version is still missing those columns

1. "long" and "lat" are the ones used by the model.
2. This should be addressed by timberid and validated in a subsequent PR @benwulfe 
3. The additional features that I see are not used by the model, so we don't need them. https://github.com/tnc-br/ddf-isoscapes/blob/main/dnn/variational_inference.ipynb. and 
![image](https://github.com/tnc-br/ddf-isoscapes/assets/137205035/6bb9f1e7-a6d7-4fc8-9801-8a1caf7379da)

This is no-op at this time. We can add a TODO for timberid to add the columns needed by the model.

Reply:
Thanks-- (2) is informative (@benwulfe ) and it's good to know we handle missing columns gracefully. Do we print a warning?

Reply:
We don't. Updated:
![image](https://github.com/tnc-br/ddf-isoscapes/assets/137205035/d936d220-02e7-44db-9613-ee73ffe987cd)